src/main/utils.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns the major version of electron
+ */
+export function getElectronVersion(): { major: number; minor: number; patch: number } {
```

Comment:
We should cache this result right? I know it’s not that expensive, but it’s reasonable to assume that it won’t change in application lifetime + it current route is hot code path right? Maybe this is over optimization lol

Reply:
Thought the same, but `process.versions.electron` is a readonly string, a "cache" would do the same.